thewalrus/tests/test_hafnian_repeated.py

```diff
@@ -115,7 +115,13 @@ def test_complex(self):
         haf = hafnian_repeated(A, [1] * 6)
         expected = jhaf(np.complex128(A), np.ones([6], dtype=np.int32))
         assert np.allclose(haf, expected)
-
+    def test_loop_true(self):
+        """Check hafnian_repeated(A)=0 if A is a zero matrix and
+        loop is true.
+        """

```

Comment:
For consistent formatting, add a blank line before the new test and use braces on code-like text.

```suggestion

    def test_loop_true(self):
        """Check `hafnian_repeated(A)=0` if `A` is a zero matrix and
        `loop` is true.
        """
```

Reply:
We will add them for formatting. Also, we need to make better docstrings for the functions that we added so we still need to work a little bit. We will message you when we think we finished them, or if we have any other questions!